install/installer/pkg/components/server/networkpolicy.go

```diff
@@ -86,6 +86,24 @@ func Networkpolicy(ctx *common.RenderContext, component string) ([]runtime.Objec
 							},
 						},
 					},
+					{
+						Ports: []networkingv1.NetworkPolicyPort{
+							{
+								Protocol: common.TCPProtocol,
+								Port:     &intstr.IntOrString{IntVal: IAMSessionPort},
+							},
+						},
+						From: []networkingv1.NetworkPolicyPeer{
+							{
+								PodSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{
+										"app":       "gitpod",
+										"component": "iam",
```

Comment:
Is it possible to use `iam.Component` as a constant here or does that cause import cycles?

Reply:
done